uxarray/grid/grid.py

```diff
@@ -347,6 +348,28 @@ def validate(self):
         else:
             raise RuntimeError("Mesh validation failed.")
 
+    def compute_face_center(self, method="average"):
+        """Constructs a face_lon and face_lat.
```

Comment:
Can we make this a bit more descriptive? Should mention that this method provides the user direct control to select what type of method is used for constructing the face centers, as opposed to the default option chosen by the grid properties.

Can also mention this can be used to override / re-compute stored/parsed values.